koa-bodyparser/koa-bodyparser.d.ts

```diff
@@ -18,6 +18,11 @@
 declare module "koa-bodyparser" {
 
     import * as Koa from "koa";
+    module "koa" {
```

Comment:
I have try to move this block outside the `declare module "koa-bodyparser"`, but it doesn't work in other middleware.
For example, I use a `koa-bodyparser` middleware and there are some code look like this:
```
const router: Router = new Router()
    .get("/", async (ctx, next) => {
        const test = ctx.request.body;
        await next();
    });
```
If I move the block outside ,then typescript will notice me that `.body` does not exist in `ctx.request`. However, when I move the block inside `declare module "koa-bodyparser"`, it works in the right way.